Source: Hong Hao <oahong@gmail.com>
Upstream: mail send
Reason: gcc 4.4
--- qstardict-0.13.1/plugins/stardict/dictziplib.cpp.orig	2009-05-17 17:48:37.605944568 +0800
+++ qstardict-0.13.1/plugins/stardict/dictziplib.cpp	2009-05-17 18:35:06.757994635 +0800
@@ -305,7 +305,7 @@
 
 #ifdef Q_OS_UNIX
     if (stat(fname.c_str(), &sb) || !S_ISREG(sb.st_mode))
-#elif def Q_OS_WIN32
+#elif defined Q_OS_WIN32
     if (_stat(fname.c_str(), &sb) || !(sb.stMode & _S_IFREG))
 #endif
     {
